### PR TITLE
Update Exchange-Teams-interact.md

### PR DESCRIPTION
**Pull request by dacarroll**
Adding note for Office 365 environments capable of supporting create and view meetings for mailboxes hosted on-premises.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -84,6 +84,9 @@ Microsoft Teams works with several Microsoft 365 and Office 365 services to prov
 
 ## Requirements to create and view meetings for mailboxes hosted on-premises
 
+  > [!NOTE]
+  > Create and view meetings for mailboxes hosted on-premises is currently only supported within commerical, GCC, and GCCH environments.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:88]:
```suggestion
  > Create and view meetings for mailboxes hosted on-premises is currently only supported within commerical, GCC, and GCC High environments.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dacarroll, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**